docs/source/guides/installation_guide.md

```diff
@@ -3,35 +3,26 @@
 
 ## Contents
 
-* [Running Ursula with Systemd](#System-Requirements)
-* [Running Ursula with Systemd](#Systemd-Service-Installation)
+* [System Requirements and Dependencies](#System-Requirements-and-Dependencies)
 * [Standard Installation](#Standard-Installation)
 * [Development Installation](#Development-Installation)
 * [Running Ursula with Systemd](#Systemd-Service-Installation)
 
-## System Requirements
+## System Requirements and Dependencies
 
-* At least 1 GB of RAM is required for key derivation functionality (SCrypt).
+* At least 1 GB of RAM is required for secure password-based key derivation with [scrypt](http://www.tarsnap.com/scrypt.html).
 * We have tested `nucypher` with Windows, Mac OS, and GNU/Linux (GNU/Linux is recommended).
+* If you don’t already have it, install [Python](https://www.python.org/downloads/). 
+As of August 2019, we are working with Python 3.6, 3.7, and 3.8.
+* We also require the following system packages (Linux):
```

Comment:
Yeah, idk what is required in which distribution, but in Ubuntu-like distros I use this:
```
sudo apt install libssl-dev build-essential automake pkg-config libtool libffi-dev libgmp-dev
```
However I am not sure if libgmp is needed (probably not)

Reply:
In ubuntu-like distros, you really need `build-essential`